Mathlib/CategoryTheory/Yoneda.lean

```diff
@@ -437,6 +437,19 @@ lemma isIso_of_yoneda_map_bijective {X Y : C} (f : X ⟶ Y)
   obtain ⟨g, hg : g ≫ f = 𝟙 Y⟩ := (hf Y).2 (𝟙 Y)
   exact ⟨g, (hf _).1 (by aesop_cat), hg⟩
 
+lemma isIso_iff_isIso_yoneda_map {X Y : C} (f : X ⟶ Y) :
```

Comment:
I would suggest adding first a variant `isIso_iff_yoneda_map_bijective`, and then deducing this one.